src/kirby/components/modal/modal.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="modal-content">
+    <h2 [ngClass]="'text-' + config.titleHorizontalAlignment">{{ config.title }}</h2>
```

Comment:
Consider simplifying:
```suggestion
    <h2 class="text-{{config.titleHorizontalAlignment}}">{{ config.title }}</h2>
```